src/controller/DevicePairingDelegate.h

```diff
@@ -40,6 +40,7 @@ class DLL_EXPORT DevicePairingDelegate
     {
         SecurePairingSuccess = 0,
         SecurePairingFailed,
+        SecurePairingContinue,
```

Comment:
I think we should call this `SecurePairingDiscoveringMoreDevices`.

Reply:
Changed.